### ISSUE TITLE
Req: Users to specify location/filename/file extension/linking of transcription files

### ISSUE THREAD

**Issue by glocalglocal** [ccbogel/QualCoder#403]
Linked to #402, it would be good to allow the user to specify the filename, filetype, location and linked/embedded status of each transcription file generated. Eg my text, AV and transcriptions are located in different folders and QC's defaults can mess up this structure.  

The filename extension is important if we need to use a particular extension for some reason (eg I use .md for markdown). If I change the filename extension set by QC to make the text file work as .md, I will have to re-link it to QC with the new filename, which adds to the workflow and can lead to user errors. 

It would also help dramatically with efficiency if every linked file also had a 'open location in file manager' option. 

Win10, QC 2.5

**Comment by ccbogel** [comment]
OK I understand. I will work on the other issues first. This will require a change to at least one database table.

**Comment by ccbogel** [comment]
OK. I think I have some workarounds here. 
When importing/linking an A/V file a database text transcription entry is created. This is automatic.
My suggestion is to create/edit your text transcription externally, then also import/or link it into QC. 
Make sure it's name is not identical to the one that QC created for the transcription, or a duplicate name error will occur.
Then on the transcription file that is currently associated with the A/V (Noted with a black T and red arrow), right click and select the option - to choose an alternative transcription file. (Pick the one you imported/linked to).
![pic1](https://user-images.githubusercontent.com/5834638/140307087-ffec6675-fa9e-4f6f-9885-1048aa82d340.png)

**Comment by glocalglocal** [comment]
I am getting this for a blank txt file that was created when I opened an mp3 file: 
![image](https://user-images.githubusercontent.com/16408198/140620824-8ebcbc27-cab9-4117-82aa-9e8e942f001b.png)

Previously, I had transcribed that mp3 file, copied the text into an .md file, and linked it to the project. Obviously, qc didn't know the .md file was associated with the mp3 file. I think there should be a way to tell qc that after the event, as your screenshot suggests, but I don't get that menu option.
